portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx

```diff
@@ -27,12 +27,13 @@ import {
   searchField,
   tableStyles,
 } from "../../Common/FormComponents/common/styleLibrary";
-import { IPVCsResponse, IStoragePVCs } from "../../Storage/types";
+import { IStoragePVCs } from "../../Storage/types";
 import { setErrorSnackMessage } from "../../../../actions";
 import { ErrorResponseHandler } from "../../../../common/types";
 import api from "../../../../common/api";
 import TableWrapper from "../../Common/TableWrapper/TableWrapper";
 import SearchIcon from "../../../../icons/SearchIcon";
+import DeletePVC from "./DeletePVC";
```

Comment:
```suggestion
import withSuspense from "../Common/Components/withSuspense";

const DeletePVC = withSuspense(
  React.lazy(() => import("./DeletePVC"))
);
```

Lazy load the modal component